docs/dev_guide/remote_data.rst

```diff
@@ -7,8 +7,8 @@ Remote Files
 Remote Data Manager
 ===================
 
-There are functions which require files to be downloaded from the internet before they are executed. If the files are changed on the remote
-server or if they are tampered with after the file is downloaded, the function may return erraneous results.
+There are functions which require files to be downloaded from the internet before they are executed.
+If the files are changed on the remote server or if they are tampered with after the file is downloaded, the function may return erraneous results.
```

Comment:
If the file is locally corrupted, do we redownload it?

Reply:
No, we raise error.

Reply:
Would it not make more sense to just redownload it?

Reply:
Now that I think about it, yes it makes more sense to redownload. I think I wrote the extra logic to check it because silently redownloading when cache doesn't match didn't make a lot of sense.

Reply:
Have you changed the code to do this? If so can you update this sentence?